### DISCUSSION TITLE
Images are not showing, please check the error in my code

### DISCUSSION THREAD

**Discussion by Rock8090** [datasciencecompleteguide/Machine-Learning#42] [Q&A]
import glob
import cv2 as cv
path = glob.glob("C:/Users/DELL/Desktop/Documents/Ritesh Sir/Pulses_Images/*.*")

cv_img = []
for img in path:
    n = cv.imread(img)
    cv_img.append(n)
    cv.imshow( cv_img)
    cv2.waitKey(0)
    cv2.destroyAllWindows()

**Comment by datasciencecompleteguide** [comment]
`path = glob.glob("C:/Users/DELL/Desktop/Documents/Ritesh Sir/Pulses_Images/.")`
`glob.glob('path/to/files/*.jpg')`
This is the exact format with the use of star.

Instead of passing the complete list to the imshow function....iterate thru the list and pass the individual images to display.

**Reply by Rock8090** [reply]
path = glob.glob("C:/Users/DELL/Desktop/Documents/Ritesh Sir/Pulses_Images/*.*")
    path = glob.glob("C:/Users/DELL/Desktop/Documents/Ritesh Sir/Pulses_Images/*.*")
i used both way but image doesn't show , please check it
